src/transformers/models/mask2former/configuration_mask2former.py

```diff
@@ -97,11 +97,11 @@ class Mask2FormerConfig(PretrainedConfig):
     attribute_map = {"hidden_size": "mask_feature_size"}
     backbones_supported = ["swin"]
     decoders_supported = ["detr"]
-    pixel_decoders_supported = ["detr"] #since deformable_detr not available in transformers yet
+    pixel_decoders_supported = ["deformable_detr"] #since deformable_detr not available in transformers yet
```

Comment:
No need for this comment anymore:
```suggestion
    pixel_decoders_supported = ["deformable_detr"] 
```